src/entity/Author.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, PrimaryGeneratedColumn, Column, BaseEntity, OneToMany } from "typeorm";
+import { Book } from "./Book.entity";
+import { COLUMN_LENGTH } from '../../constants';
+
+@Entity()
+export class Author extends BaseEntity {
```

Comment:
```suggestion
export class Author extends BaseEntity {
  constructor(data?: Partial<Author>) {
    data && Object.assign(this, data);
  }
```
Bổ sung thêm hàm tạo cho enity để sau build đối tượng nhanh hơn e nha
VD: 
```
    new Author({
        family_name: "Vy",
        family_name: "Nguyen",
        ...
    })
```

Reply:
dạ em chỉnh rùi ạ